auth.go

```diff
@@ -62,6 +67,42 @@ func (a *Auth) CheckPermission(scope, userID string) error {
 	return ErrNotAllowed
 }
 
+// GetUsers returns a list of userIDs having one or more permission scopes
+func (a *Auth) GetUsers() ([]string, error) {
```

Comment:
What do you think about renaming this to `Users`?

https://golang.org/doc/effective_go.html#Getters

Reply:
I'm easy either way. Though feel this function isn't really a getter in the sense of getting a field value as described in the effective go section you linked.

I've updated the function names as per your suggestion in https://github.com/go-joe/joe/pull/28/commits/2babcb8d6b43e6663aa39f58f24b9ba5317009f4